static/app/api.tsx

```diff
@@ -601,6 +606,7 @@ export class Client {
             statusText,
             responseJSON,
             responseText,
+            headers: Object.fromEntries(response.headers.entries()),
```

Comment:
note: looking at the current usage of `getResponseHeader`, we only ever extract those:

- 'Link'
- 'X-Hits'
- 'Endpoint'
- 'X-Sentry-Direct-Hit'
- 'X-Max-Hits'

Not sure if reading all 30+ headers of every response is overkill here, but I think it shouldn’t matter much.